fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/FeltPrivacyInfoCard.kt

```diff
@@ -36,7 +37,9 @@ fun FeltPrivacyModeInfoCard(
 ) {
     Card(
         modifier = Modifier
-            .wrapContentHeight(),
+            .wrapContentHeight()
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp),
```

Comment:
This will actually cause a horizontal padding of 32dp because we set the 16dp horizontal padding in the view @ https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt#50

You will be fine visually by removing this `padding(horizontal = 16.dp)` otherwise you will have extra padding that wasn't intended.

T configuration change bug is pretty well known for our homepage. The current known fix for it is to convert the recyclerview into compose.

Reply:
ah I removed the extra padding. thanks!